src/Security/Security/Cmdlets/Alerts/GetAlerts.cs

```diff
@@ -26,6 +27,8 @@ namespace Microsoft.Azure.Commands.Security.Cmdlets.Alerts
     [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SecurityAlert", DefaultParameterSetName = ParameterSetNames.SubscriptionScope), OutputType(typeof(PSSecurityAlert))]
     public class GetAlerts : SecurityCenterCmdletBase
     {
+        private const int MaxAlertsToFetch = 1500;
```

Comment:
Instead of setting an arbitrary maximum, you should follow the SupportsPaging pattern, so the user can specify where the list should start and end.  Youc an see an example of thsi pattern here:  https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/ActiveDirectory/Cmdlets/GetAzureADUserCommand.cs

Reply:
The reason i'm using this maximum number is that the RP does not support the Skip and First parameters

Reply:
@markcowl, this is our long-term plan, across all Microsoft.Security PS API.

This change is just a temporally one, as we have customers complaining about this specific API and the better solution is much more expansive for us for now.

Reply:
You can support paging without support for First and Skip in your API, there are examples of this in the repo.  Since this is a preview module however, the quick fix is OK for now.  You will need to fix this before inclusion in the 'Az' module